tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/Cacher.scala

```diff
@@ -5,6 +5,8 @@ import at.forsyte.apalache.tla.lir._
 import at.forsyte.apalache.tla.lir.transformations.{TlaExTransformation, TlaModuleTransformation, TransformationTracker}
 import at.forsyte.apalache.tla.lir.UntypedPredefs._
 
+// TODO: Remove?
+
```

Comment:
what is the role of this `Cacher`? Do we still need it? If not, let's remove it.

Reply:
I think it was something we disabled a long time ago, I just found it while scanning for instances of OperDecl ctors. If we do remove it, however, that should be its own PR.